src/templates/thanks/form-data-outlook.html

```diff
@@ -74,7 +74,7 @@
                                             <li>{{t('name')}}: {{witness['name'] or 'N/A' }}</li>
                                             <li  class="nocapitalize">{{t('email')}}: <a href='mailto:{{witness['email'] or ';'}}'>{{witness['email'] or 'N/A' }}</a></li>
                                             <li>{{t('phone')}}: {{witness['phoneNumber'] or 'N/A' }}</li>
-                                            <li  class="nocapitalize">{{t('additional_information')}}: {{witness['anythingElse'] or 'N/A' }}</li>
+                                            <li  class="nocapitalize">{{t('follow_up')}}: {{witness['anythingElse'] or 'N/A' }}</li>
```

Comment:
Are you sure additional_information on the witness page should be changed to "follow_up"?

Reply:
No, I think I goofed that. 
In rolling back that change I noticed that the thank you template may not be consistent for outlook vs non-outlook templates.